_rules/element-marked-decorative-is-not-exposed-46ca7f.md

```diff
@@ -50,44 +50,44 @@ When these conflicts arise on [decorative][] [non-text content][], this is also
 
 #### Passed Example 1
 
-This `img` element is [marked as decorative][] through its `alt` attribute and is not [included in the accessibility tree][] because of the `aria-hidden` attribute.
+This `img` element is [marked as decorative][] through its empty `alt` attribute and has [semantic role][] of `none`.
 
 ```html
-<img src="/test-assets/shared/w3c-logo.png" alt="" aria-hidden="true" />
+<img src="/test-assets/shared/w3c-logo.png" alt="" />
 ```
 
 #### Passed Example 2
 
-This `img` element is [marked as decorative][] through its `alt` attribute and is not [included in the accessibility tree][] because it is `hidden` to everyone.
+This `img` element is [marked as decorative][] through its `alt` attribute and is not [included in the accessibility tree][] because of the `aria-hidden` attribute.
 
 ```html
-<img src="/test-assets/shared/w3c-logo.png" alt="" hidden />
+<img src="/test-assets/shared/w3c-logo.png" alt="" aria-hidden="true" />
 ```
 
 #### Passed Example 3
 
-This `nav` element is [marked as decorative][] through its `role` attribute and has a [semantic role][] of `none`.
+This `img` element is [marked as decorative][] through its `alt` attribute and is not [included in the accessibility tree][] because it is `hidden` to everyone.
```

Comment:
```suggestion
This `img` element is [marked as decorative][] through its empty `alt` attribute and is not [included in the accessibility tree][] because it is `hidden` to everyone.
```